config.json

```diff
@@ -168,7 +168,7 @@
         "name": "Resistor Color Duo",
         "uuid": "de800041-3dcc-41b9-b101-7314ff685c93",
         "practices": [],
-        "prerequisites": [],
+        "prerequisites": ["arrays"],
```

Comment:
Maybe better: array-transformations?

Reply:
Could you explain what you mean by "array-transformations" and "array-analysis", I don't get the subtlety

Reply:
They are concepts, with existing exercises :D.

- analysis: https://github.com/exercism/javascript/blob/main/exercises/concept/elyses-analytic-enchantments/.docs/introduction.md
- transformations: https://github.com/exercism/javascript/blob/main/exercises/concept/elyses-transformative-enchantments/.docs/introduction.md

Reply:
Got it :ok_hand:

Reply:
I feel like this would be better to put "array-analysis" just like in "Resistor Color". IMO this could also be a good exercise to add "array-destructuring", what do you think ?

Reply:
Sounds good @TomPradat